src/code/z_actor.c

```diff
@@ -4432,13 +4432,26 @@ void Animation_ChangeByInfo(SkelAnime* skelAnime, AnimationInfo* animationInfo,
                      frameCount, animationInfo->mode, animationInfo->morphFrames);
 }
 
-void func_80034F54(PlayState* play, s16* arg1, s16* arg2, s32 arg3) {
+/**
+ * Fills two tables with rotation angles that can be used to simulate idle animations.
+ *
+ * The rotation angles are dependent on the current frame, so should be updated regularly, generally every frame.
+ *
+ * This is done for the desired limb by taking either the `sin` of the yTable value or the `cos` of the zTable value,
+ * multiplying by some scale factor (generally 200), and adding that to the already existing rotation.
+ *
+ * Note: With the common scale factor of 200, this effect is practically unnoticeable if the current animation already
+ * has motion involved.
+ *
+ * Note: MM gets this function unused in favor of `SubS_UpdateFidgetTables` @ `z_sub_s.c`.
```

Comment:
```suggestion
 * Note: In MM this function is unused in favor of `SubS_UpdateFidgetTables` @ `z_sub_s.c`.
```

Although I don't know if this comment is necessary

Reply:
I dont think its worth commenting on at all. If we commented on everything MM changed it would get unruly fast.
And SubS has no meaning in OoT at all, so it would just be more confusing imo.

Reply:
right, removed then: 389d3bcdb06f3848221c8ebb8bae0581ddfe54a5